controllers/apply-next/views/questions-list.njk

```diff
@@ -0,0 +1,110 @@
+{% macro fieldTypeHelpText(field, step) %}
+    {% if field.type === 'checkbox' %}
+        Pick one of:
+    {% elseif field.type === 'radio' %}
+        Choose as many as you like from:
+    {% elseif field.type === 'budget' %}
+        Provide an itemised budget
+        {% if field.attributes.max %}
+            (to a maximum total of £{{ field.attributes.max.toLocaleString() }}{% if field.attributes.rowLimit %} across {{ field.attributes.rowLimit }} items{% endif %})
+        {% endif %}
+    {% elseif field.type === 'currency' %}
+        Provide a financial figure
+    {% elseif field.type === 'address' %}
+        Provide an address
+    {% elseif field.type === 'date' or field.type === 'day-month' %}
+        Provide a date
+    {% elseif field.type === 'email' %}
+        Provide an email address
+    {% elseif field.type === 'file' %}
+        Upload a file
+    {% elseif field.type === 'number' %}
+        Provide a number
+    {% elseif field.type === 'tel' %}
+        Provide a telephone number
+    {% elseif field.type === 'text' %}
+        Provide a sentence or two
+    {% elseif field.type === 'textarea' %}
+        Provide a longer answer
+        {% if field.settings.minWords and field.settings.maxWords %}
+            (between {{ field.settings.minWords }}-{{ field.settings.maxWords }} words)
+        {% else %}
+            {% if field.settings.minWords %} (minimum of {{ field.settings.minWords }} words){% endif %}
+            {% if field.settings.maxWords %} (maximum of {{ field.settings.maxWords }} words){% endif %}
+        {% endif %}
+    {% endif %}
+    {% if step.notRequired %}
+        &mdash; optional, but may be required depending on your circumstances
+    {% elseif field.isRequired %}
+        &mdash; required
+    {% endif %}
+{% endmacro %}
+
+{% macro header() %}
+    <div class="u-tone-background-tint u-padded">
+        {{ caller() }}
+    </div>
+{% endmacro %}
+
+{% macro question(first = false) %}
+    <div class="{% if not first %}u-padded-top u-border-top--thin u-margin-top{% endif %}">
+        {{ caller() }}
+    </div>
+{% endmacro %}
+
+{% macro questionList(form) %}
```

Comment:
I'd be tempted to style this up using BEM class names rather than lots of utility classes. Not sure if this is just to get something on the page for now?

Reply:
Do you think it's better for this stuff to have its own classnames that are just duplicating those utility classes underneath? Happy to rewrite them all if that's the case, but otherwise why have the `u-` classes?

Reply:
My general school of thought on this is that if something can be named as a concept I'd err towards dedicated BEM classes. And to use utility classes mostly for exceptions. I could go either way in this case as it's not using _mountains_ of utility classes to achieve this layout.

Reply:
Have tried to address this – see latest commits.